addons/medical/functions/fnc_clearExpired.sqf

```diff
@@ -0,0 +1,33 @@
+/*
+ * Author: Orbis2358
+ * Remove time expired actions from list
+ *
+ * Arguments:
+ * 0: The patient <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [patient] call ace_medical_fnc_clearExpired
+ *
+ * Public: Yes
+ */
+
+#include "script_component.hpp"
+
+params ["_target"];
+
+private _joint = _target getVariable [QGVAR(jointTreatment), []];
+private _itemCount = 0;
+
+if (!(_joint isEqualTo [])) then {
+    {
+        if ((_x select 3) < CBA_missionTime) then {
```

Comment:
you can use `select` to remove them.

Reply:
Yeah, I was pretty dumb for some reason while coding that file. Thanks for letting me know XP